src/test-integration/studio/StudioTest.java

```diff
@@ -17,6 +17,7 @@
 
     @Override
     protected void onSetUp() {
+        System.setProperty("user.home", "/tmp/");
```

Comment:
Why is this necessary? The current version is not failing. Also if this is absolutely necessary, just providing /tmp is a bad idea as apps may end up littering it and a different user who tries to create the same file will get a permission error. So at the minimum I include the user name after /tmp or /var/tmp.

Reply:
Updated to create a new tmp dir on each run